src/main/java/org/commcare/formplayer/objects/SerializableMenuSession.java

```diff
@@ -1,30 +1,61 @@
-package org.commcare.formplayer.repo;
+package org.commcare.formplayer.objects;
 
 import org.commcare.formplayer.session.MenuSession;
+import org.hibernate.annotations.GenericGenerator;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import javax.persistence.*;
 
 /**
  * Created by willpride on 8/1/16.
  */
+
+@Entity
+@Table(name="menu_sessions")
+@EntityListeners(AuditingEntityListener.class)
```

Comment:
this isn't needed unless you're going to add a creation / modification date. I do think a creation date would be useful and would allow us to clean up this data better.